data/data.go

```diff
@@ -40,12 +41,27 @@ func (y EffxYaml) newConfig() (*effx_api.ConfigurationFile, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	metadata, err := inferMetadata(filepath.Dir(y.FilePath))
+	if err != nil {
+		log.Printf("Could not predict version %+v\n", err)
+	}
+
 	config.FileContents = string(yamlFile)
 	config.SetAnnotations(map[string]string{
 		"effx.io/source":    "effx-cli",
 		"effx.io/file-path": y.FilePath,
 	})
 
+	if metadata != nil {
+		config.SetAnnotations(map[string]string{
+			"effx.io/cli-inferred-tags": metadata.lang,
+		})
+		config.SetTags(map[string]string{
+			metadata.lang: metadata.version,
```

Comment:
will this also set `language: <lang>` somewhere? we should be doing both, everywhere when possible

Reply:
can we make sure we are not overwriting someones tags/annotations? does that already happen?

Reply:
good idea, setting langugage as a tag.

also I added something to ensure we are only appending to tags annotations, we won't wipe anything